### ISSUE TITLE
build for linux-aarch64

### ISSUE THREAD

**Issue by alfu32** [Jarred-Sumner/bun-releases-for-updater#1]
hello,

would it be possible to also publish a build for linux aarch64 ?

thank you.

**Comment by adamjosefus** [comment]
Same.

`Failed to download bun from https://github.com/Jarred-Sumner/bun-releases-for-updater/releases/latest/download/bun-linux-aarch64.zip`

The URL return 404. Probably bad assign assets in github release.

**Comment by holiq** [comment]
Yes you are right, there is no file for linux-aarch64 on release
